fluent_metrics.conf.rt

```diff
@@ -422,7 +422,7 @@
   tag jfrog.rt.artifactory.access
   <parse>
     @type regexp
-    expression /^(?<timestamp>[^ ]*) \[(?<trace_id>[^\]]*)\] \[(?<action_response>[^\]]*)\] (?<repo_path>.*) for client : (?<username>.+)/(?<ip>.+)\.$/m
+    expression /^(?<timestamp>[^\s]*) \[(?<trace_id>[^\]\s]*)\] \[(?<action_response>[^\]\r\n]*)\] (?<repo_path>[^\s]*)? *for client : *(?<username>[^\s]*) *\/ *(?<ip>[^\s]*)?[\.\s]?(\[(?<token>[^\s]*)\])?$/m
```

Comment:
Not a blocker. Just a suggestion for future. In previous job where lots of regexes were used, we had a team rule that any regex must be accompanied by examples of string that the regex tries to match, as well as comprehensive tests. Obviously tests aren't possible here but I think examples here would help future us.

Reply:
great point. I'll add some examples